esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -177,19 +186,35 @@ define([
 
         _onUpload: function (event) {
             var context = this;
+            var targetRow;
             if (!this.dropZoneTargetSelect.initalized) {
                 this.dropZoneTargetSelect.init({
                     DropZones: true,
                     callback: function (value, row) {
+                        if (context.dropZoneMachineSelect) {
+                            context.dropZoneMachineSelect.defaultValue = context.dropZoneMachineSelect.get("value");
+                            context.dropZoneMachineSelect.loadDropZoneMachines(value);
+                            targetRow = row;
+                        }
+                    }
+                });
+            }
+
+            if (!this.dropZoneMachineSelect.initalized) {
+                this.dropZoneMachineSelect.init({
+                    DropZoneMachines: true,
+                    callback: function (value, row) {
+                        targetRow.machine.Name = value
+                        targetRow.machine.Netaddress = value
                         if (context.dropZoneFolderSelect) {
-                            context.dropZoneFolderSelect.reset();
```

Comment:
After various testing, the .reset() did not actually clear out the values on the drop down. So, I do not believe it was needed.